App/features/photos/sagas.ts

```diff
@@ -41,48 +75,40 @@ function * queryForNewPhotos() {
       const results: LocalPhotoResult[] = yield call(requestLocalPhotos, since)
       yield put(actions.queryCameraRoll.success(results))
       yield put(actions.updateLastQueriedTime(currentRefresh))
-      for (const localPhotoResult of results) {
-        yield fork(preparePhoto, localPhotoResult.assetId)
-      }
+      yield all(
+        results.map((result) =>
+          put(addTaskChannel, { payload: { id: result.assetId } })
+        )
+      )
```

Comment:
Here we now just add any new photo tasks to the queue instead of processing them directly